app/dmc/src/main.c

```diff
@@ -230,7 +230,7 @@ static int bh_chip_run_smbus_tests(struct bh_chip *chip)
 	int pass_val = 0xFEEDFACE;
 	uint8_t count;
 	uint8_t data[32]; /* Max size of SMBUS block read */
-
+	uint32_t app_version;
```

Comment:
Sort of surprised that this wasn't caught by the linter, but can you keep the newline under this?